cylc/flow/task_remote_mgr.py

```diff
@@ -547,7 +547,7 @@ def _file_install_callback(self, ctx, platform, install_target):
                     )
         if ctx.ret_code == 0:
             # Both file installation and remote init success
-            LOG.debug(f"File installation complete for {install_target}")
+            log_platform_event('remote file install complete', platform)
```

Comment:
Can we keep this at debug level:

```suggestion
            log_platform_event('remote file install complete', platform, level=logging.DEBUG)
```

(requires `import logging` at top of file)

Reply:
Yes, happy to do this. I know at one stage we were keen on being able to work out the time taken by remote install from the logs. I take it we are happy for that only to be in debug mode?

Reply:
I can't remember anything about recording remote-init times although they can be worked backwards from preparing times.

Reply:
Happy to leave.